.gitignore

```diff
@@ -150,3 +150,6 @@ dist/
 /guix-build-*
 
 /ci/scratch/
+
+# Jetbrains
+.idea
```

Comment:
This should be done in developer's personal environment, not in the repo.

Reply:
Ah yeah, there is a global gitignore. Thanks, will revert.